### PR TITLE
Add recipe for helppc/supervisor-bundle

### PR DESCRIPTION
**Pull request by tomas-kulhanek**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by tomas-kulhanek** [comment]
This package can use with Symfony http client  or Guzzle. What can i do for fix Travis error?

File: helppc/supervisor-bundle/1.0/config/routes/helppc_supervisor.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+supervisor:
+    resource: "@SupervisorBundle/Resources/config/routing.xml"
```
Comment (Nyholm) [inline helppc/supervisor-bundle/1.0/config/routes/helppc_supervisor.yaml:2]:
```suggestion
    resource: '@SupervisorBundle/Resources/config/routing.xml'
```

File: helppc/supervisor-rest-bundle/1.0/config/routes/helppc_supervisor_rest.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+supervisor:
+    resource: "@SupervisorRestBundle/Resources/config/routing.xml"
```
Comment (Nyholm) [inline helppc/supervisor-rest-bundle/1.0/config/routes/helppc_supervisor_rest.yaml:2]:
```suggestion
    resource: '@SupervisorRestBundle/Resources/config/routing.xml'
```

**Review by Nyholm — COMMENTED** [review]
Thank you. Just a minor thing

File: helppc/supervisor-rest-bundle/1.0/config/routes/helppc_supervisor_rest.yaml (hunk 2 above)
Comment (tomas-kulhanek) [reply helppc/supervisor-rest-bundle/1.0/config/routes/helppc_supervisor_rest.yaml]: ouch.. thanks

**Comment by tomas-kulhanek** [comment]
@Nyholm  please how is possible to re-run travis?

**Comment by Nyholm** [comment]
You cannot. And I could not restart it myself...

You could make a new commit: 

```
git commit --allow-empty
git push
```

**Comment by tomas-kulhanek** [comment]
@Nyholm  thanks. thats new information for me :)

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**